### PR TITLE
Update README screenshots for #1667

### PR DESCRIPTION
**Pull request by matt852**
# Closes: #1667
# What's Changed

When I view the current screenshots on the README landing page, I do not feel like they accurately show all of Nautobot's capabilities, benefits, or really differentiate it from Netbox.  The current screenshots are from Nautobot v1.0.0b1, and there are significant changes since then that I think should be featured.

I went ahead and updated them to more accurately reflect Nautobot's current state of development, as well as highlight some great new features it offers.  The updated screenshots are from v1.2.9 and show base functionality as well as some plugin benefits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,27 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
 
 ### Screenshots
 
-![Screenshot of main page](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot1.png "Main page")
+![Screenshot of main page](nautobot/docs/media/screenshot1.png "Main page")
```
Comment (glennmatthews) [inline README.md:23]: IIRC these need to be absolute links for the README to render correctly on PyPI (i.e. https://pypi.org/project/nautobot/)

File: README.md [hunk 2]
```diff
@@ -20,15 +20,27 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
...
+![Screenshot of config contexts](nautobot/docs/media/screenshot2.png "Config Contexts")
 
 ---
 
-![Screenshot of prefix hierarchy](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot3.png "Prefix hierarchy")
+![Screenshot of prefix hierarchy](nautobot/docs/media/screenshot3.png "Prefix hierarchy")
+
+---
+
+![Screenshot of golden config](nautobot/docs/media/screenshot4.png "Golden config")
```
Comment (glennmatthews) [inline README.md:35]: Not sure it's ideal to put screenshots of plugins in the main repo's README... maybe under a separate subheading `#### Plugin Screenshots`? Not sure.

File: README.md (hunk 1 above)
Comment (matt852) [reply README.md:23]: Ah ok, thanks I'll revert them.

File: README.md (hunk 2 above)
Comment (matt852) [reply README.md:35]: I think that could work too.  I was trying to think of a way to showcase it's extensibility, but separating them out into a separate section might be a better idea.

Comment (matt852) [reply README.md]: Updated into a separate section

**Comment by matt852** [comment]
I made a mistake with the merging in of the `develop` branch.  I'm going to re-branch this, and submit a new PR with proper commits.

**Pull request closed without merging**